H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs

```diff
@@ -54,16 +53,16 @@ public partial class ServerBrowserViewModel : ObservableObject
 
     [ObservableProperty]
     private ServerTabViewModel _selectedTab;
-
     private ServerTabViewModel AllServersTab { get; set; }
     private ServerTabViewModel FavouritesTab { get; set; }
     public ObservableCollection<ServerTabViewModel> ServerTabs { get; set; } = [];
 
     [ObservableProperty]
     private ServerFilterViewModel _advancedServerFilter = new();
 
+    [ObservableProperty]
+    private PasswordViewModel _passwordViewModel = new();
```

Comment:
Rather instantiate a new one every time you show the dialog, to not keep the password around in memory

Reply:
Agree